include/robot_state_publisher/robot_state_publisher.h

```diff
@@ -71,20 +72,20 @@ class RobotStatePublisher
   /** Constructor
    * \param tree The kinematic model of a robot, represented by a KDL Tree
    */
-  RobotStatePublisher(const KDL::Tree& tree, const urdf::Model& model = urdf::Model());
+  explicit RobotStatePublisher(const KDL::Tree& tree, urdf::Model  model = urdf::Model());
```

Comment:
I don't think changing the constref to an object is what we want; that will cause the copy constructor to be called.